js/controllers/form/MultilingualInputHandler.js

```diff
@@ -247,6 +246,7 @@
 			// html source, for example (both actions open a new window).
 			if ($(tinyMCEObject.getContainer()).find('iframe').attr('id') ==
 					$(document.activeElement).attr('id')) {
+				$(document.activeElement).find('div.mce-tinymce-inline').hide();
```

Comment:
What's going on in this code? I can see you're hiding the inline editor, but as far as I can tell, that's when the field is still the active element. I must be misunderstanding something here so just looking for some clarity.

Reply:
See my comments here: https://github.com/pkp/pkp-lib/issues/2564#issuecomment-377704207
In short that is an attempt to solve the problem with the multilingual forms. Have to say that I did not check yet whether it breaks the single locale title field.

Reply:
Now that I look at it, I have no idea 🤣 

But I removed that and added this: https://github.com/pkp/pkp-lib/pull/3544/files#diff-83df7d7f7c760e14fa91323920849101R235

The problem was that the editor appeared/disappeared as expected, however clicking some button (bold) closed the multilingual popover. So I made the closing js there conditional.

Worked for me, can you check that out?